src/components/staking/PageValidator.vue

```diff
@@ -109,29 +113,47 @@
         </li>
         <li>
           <h4>Validator Since</h4>
-          <span>Block #{{ validator.startHeight || 0 }}</span>
+          <span v-if="validator.startHeight"
+            >Block #{{ validator.startHeight }}</span
+          >
+          <span v-else>{{ validator.startHeight | noBlanks }}</span>
         </li>
         <li>
           <h4>Uptime</h4>
-          <span id="page-profile__uptime">
-            {{ validator.uptimePercentage | percent }}
-          </span>
+          <span v-if="validator.uptimePercentage" id="page-profile__uptime">{{
+            validator.uptimePercentage | percent
+          }}</span>
+          <span v-else id="page-profile__uptime">{{
```

Comment:
there is a bunch of douplicated code in this PR. can this be handled differently?

Reply:
Yes, you are right. I just changed it. The only issue was using filters inside a function. But SO saved me (again) :smile: